util/Random.h

```diff
@@ -67,7 +71,11 @@ inline consteval int32_t RandIntCx(const int32_t min, const int32_t max,
     return min + static_cast<int32_t>(val_in_range);
 }
 inline consteval int32_t RandIntCx(const int32_t min, const int32_t max, const int16_t n,
-                                   const std::array<char, 9> seed = std::array<char, 9>{__TIME__}) noexcept
+#ifdef STATIC_RANDOM_TESTS
```

Comment:
why not using a `#define TEST_TIME_SEED [...]` to avoid putting ifdef multiple times in the source code ?